lib/puppet/provider/service/windows.rb

```diff
@@ -56,6 +56,13 @@ def enabled?
   end
 
   def start
+    if status == :paused
+      Puppet::Util::Windows::Service.resume(@resource[:name])
```

Comment:
I double-checked that paused and disabled services can be resumed, so it is safe to place this code here.